packages/teams-js/src/public/authentication.ts

```diff
@@ -341,13 +341,12 @@ export namespace authentication {
     }
   }
 
-  
   /**
    * Different browsers handle authentication flows in pop-up windows differently.
    * Firefox and Safari, which use Quantum and WebKit browser engines respectively, block the use of 'window.open' for pop-up windows.
    * Any chrome-based browser (Chrome, Edge, Brave, etc.) opens a new browser window without any user-prompts.
    * To ensure consistent behavior across all browsers, consider using the following function to create a new authentication window.
-   * 
+   *
```

Comment:
This change doesn't necessarily need to be here?

Reply:
If he merges with my most recent change it will resolve away anyway. VSCode automatically fixes this whitespace problem so if we keep not checking it in then everyone will keep getting it put automatically into their change. 😁